Client/Frontend/Reader/ReadabilityService.swift

```diff
@@ -22,8 +22,9 @@ class ReadabilityOperation: Operation {
     var result: ReadabilityOperationResult?
     var tab: Tab!
     var readerModeCache: ReaderModeCache
+    private var rewards: BraveRewards?
```

Comment:
unfortunate that we have to pass rewards here only to configure a tab,
we can think of refactoring it later